src/proto/mod.rs

```diff
@@ -1,18 +1,29 @@
 //! Construction of messages from protobuf buffers.
+
 pub mod message;
 
-use agreement::AgreementMessage;
-use broadcast::BroadcastMessage;
+use std::fmt::{self, Debug, Display, Formatter};
+
 use merkle::proof::{Lemma, Positioned, Proof};
-use proto::message::*;
 use ring::digest::Algorithm;
-use std::fmt;
+
+use agreement::AgreementMessage;
+use broadcast::BroadcastMessage;
+use proto::message::{
+    AgreementProto,
+    BroadcastProto,
+    EchoProto,
+    LemmaProto,
+    ProofProto,
+    ReadyProto,
+    ValueProto
+};
```

Comment:
Is rustfmt okay with this? (We're using `rustfmt 0.4.2-nightly (dd807e2 2018-04-18)` and would like to enforce it in Travis, but last time I tried, a bug that I could not reproduce locally was preventing it.)

Reply:
Oh, yea. Rustfmt complains about multi-line imports. Rustfmt will ask you to line wrap at `max_width`, so assuming max_width defaults to 100, it should look like:

```
use proto::message::{
    AgreementProto, BroadcastProto, EchoProto, LemmaProto, ProofProto, ReadyProto, ValueProto,
};
```

I can change the imports to work with rustfmt 0.4.2-nightly.